perf/benchmark/run_benchmark_job.sh

```diff
@@ -228,36 +167,44 @@ trap exit_handling EXIT
 echo "Start running perf benchmark test, data would be saved to GCS bucket: ${GCS_BUCKET}/${OUTPUT_DIR}"
 
 # enable flame graph
-enable_perf_record
+#enable_perf_record
 
+DEFAULT_CR_PATH="${ROOT}/istio-install/istioctl_profiles/default.yaml"
 # For adding or modifying configurations, refer to perf/benchmark/README.md
 CONFIG_DIR="${WD}/configs/istio"
 
-for f in "${CONFIG_DIR}"/*; do
-    fn=$(basename "${f}")
-
-    # pre run
-    if [[ "${fn}" =~ "none" ]];then
-        prerun_none
-    elif [[ "${fn}" =~ "telemetryv2" ]];then
-        prerun_v2_nullvm
-    elif [[ "${fn}" =~ "plaintext" ]]; then
-        prerun_plaintext
+for dr in "${CONFIG_DIR}"/*; do
+    pushd "${dr}"
+    # install istio with custom overlay
+    if [ -e "./installation.yaml" ]; then
+       extra_overlay="-f ${dr}/installation.yaml"
     fi
+    pushd "${ROOT}/istio-install/tmp"
+      ./istioctl manifest apply -f "${DEFAULT_CR_PATH}" "${extra_overlay}" --force
+    popd
```

Comment:
we shouls wait until everyting is ready.

Reply:
ok added